mne_bids/read.py

```diff
@@ -77,37 +79,27 @@ def _read_raw(raw_fname, electrode=None, hsp=None, hpi=None, config=None,
     return raw
 
 
-def read_raw_bids(bids_fname, bids_root, return_events=True,
-                  verbose=True):
+def read_raw_bids(bids_fname, bids_root, verbose=True):
```

Comment:
I think you need to deprecate ...

Reply:
I actually never did a deprecation ... can you point me to a list of steps that need to be done? And perhaps some additional documentation on "why, how, ..." of deprecation?

Reply:
you need to deprecate whenever there is likelihood of breaking user's code. Then you warn them for one release cycle and then change it in the next cycle. Of course, we sometimes break our own rules and don't deprecate if it's not worth the effort but I think it would be good practice for you + it's good practice in general :)

Reply:
mmh I see the point in general, but I think `read_raw_bids` is so WIP / recent / alpha, that we can do it without a deprecation 🤔 

regarding good practice for me: I could read it up regardless of actually doing it :-)

You can make the final call: deprecation or not?